src/document/DocumentAnnotator.scss

```diff
@@ -11,15 +11,41 @@
         pointer-events: none;
     }
 
-    &.ba-is-highlighting {
+    &.ba-discoverability-enabled {
         .textLayer {
-            > span {
-                @include ba-HighlightCreator-cursor;
+            pointer-events: none; // Ensures that click events are received on the create region layer
+
+            span {
+                pointer-events: auto; // Text layer still receives text highlight events
+            }
+        }
+
+        &.ba-is-create--highlight {
+            .textLayer {
+                > span {
+                    @include ba-HighlightCreator-cursor;
+                }
+
+                ::selection {
+                    background-color: rgba($bdl-yellow, .33);
+                }
             }
+        }
 
-            ::selection {
-                background-color: rgba($bdl-yellow, .33);
+        &.ba-is-create--region {
+            .textLayer > span {
+                pointer-events: none;
             }
         }
     }
+
+    .textLayer {
```

Comment:
If this represents the existing styles, this should still be behind the `&.ba-is-create--highlight` class